cmd/jag/commands/config.go

```diff
@@ -5,20 +5,30 @@
 package commands
 
 import (
+	"os"
+
 	"github.com/spf13/cobra"
 	"github.com/toitlang/jaguar/cmd/jag/directory"
 )
 
+const (
+	WifiCfgKey         = "wifi"
+	WifiSSIDCfgKey     = "ssid"
+	WifiPasswordCfgKey = "pass"
+)
+
 func ConfigCmd() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:    "config",
-		Short:  "Configure Jaguar",
-		Hidden: true,
+		Use:   "config",
+		Short: "Configure Jaguar",
 	}
 
 	cmd.AddCommand(
```

Comment:
Is there a way of adding multiple sub-commands with one call? I think we use that in other places.

Reply:
done.